### PR TITLE
[Rule Tuning] Missing MITRE ATT&CK Mappings

### PR DESCRIPTION
**Pull request by terrancedejesus**
## Issues
* https://github.com/elastic/detection-rules/issues/1987

## Summary
A few mitre att&ck mappings are missing from several detection rules and should be added. We should go through these and attempt to find a proper mapping and if not detail why and skip.

<details>

- [x] Web Application Suspicious Activity: POST Request Declined
- [x] Web Application Suspicious Activity: Unauthorized Method
- [x] Web Application Suspicious Activity: No User Agent
- [x] Web Application Suspicious Activity: sqlmap User Agent
- [x] EggShell Backdoor Execution
- [x] Threat Intel Filebeat Module (v8.x) Indicator Match
- [x] Threat Intel Indicator Match
- [x] AWS IAM User Addition to Group
- [x] AWS RDS Snapshot Export
- [x] AWS RDS Snapshot Restored
- [x] AWS EventBridge Rule Disabled or Deleted
- [x] Spike in AWS Error Messages
- [x] Rare AWS Error Code
- [x] Unusual City For an AWS Command
- [x] Unusual Country For an AWS Command
- [x] Unusual AWS Command for a User
- [x] AWS Security Group Configuration Change Detection
- [x] AWS RDS Instance Creation
- [x] AWS Redshift Cluster Creation
- [x] AWS Route Table Created
- [x] AWS Route Table Modified or Deleted
- [x] Azure Automation Runbook Deleted
- [x] Azure Kubernetes Pods Deleted
- [x] Azure Virtual Network Device Modified or Deleted
- [x] Endpoint Security
- [x] GCP Storage Bucket Configuration Modification
- [x] GCP Virtual Private Cloud Network Deletion
- [x] GCP Virtual Private Cloud Route Creation
- [x] GCP Virtual Private Cloud Route Deletion
- [x] Application Added to Google Workspace Domain
- [x] Domain Added to Google Workspace Trusted Domains
- [x] Google Workspace Admin Role Deletion
- [x] Google Workspace MFA Enforcement Disabled
- [x] Google Workspace Password Policy Modified
- [x] MFA Disabled for Google Workspace Organization
- [x] Microsoft 365 Exchange DKIM Signing Configuration Disabled
- [x] Microsoft 365 Teams Custom Application Interaction Allowed
- [x] Attempt to Deactivate an Okta Network Zone
- [x] Attempt to Delete an Okta Network Zone
- [x] Attempt to Deactivate an Okta Application
- [x] Attempt to Deactivate an Okta Policy
- [x] Attempt to Deactivate an Okta Policy Rule
- [x] Attempt to Delete an Okta Application
- [x] Attempt to Delete an Okta Policy
- [x] Attempt to Delete an Okta Policy Rule
- [x] Attempt to Modify an Okta Application
- [x] Attempt to Modify an Okta Network Zone
- [x] Attempt to Modify an Okta Policy
- [x] Attempt to Modify an Okta Policy Rule
- [x] Modification or Removal of an Okta Application Sign-On Policy
- [x] Threat Detected by Okta ThreatInsight
- [x] Potential DNS Tunneling via Iodine
- [x] Netcat Network Activity
- [x] Nping Process Activity
- [x] Unusual Process Execution - Temp
- [x] Potential Microsoft Office Sandbox Evasion
- [x] Potential Persistence via Atom Init Script Modification
- [x] Unusual Hour for a User to Logon
- [x] Unusual Source IP for a User to Logon from
- [x] Rare User Logon
- [x] Spike in Failed Logon Events
- [x] Spike in Logon Events
- [x] Spike in Logon Events from a Source IP
- [x] Spike in Firewall Denies
- [x] Spike in Network Traffic
- [x] Anomalous Linux Compiler Activity
- [x] Unusual Linux Process Calling the Metadata Service
- [x] Unusual Linux User Calling the Metadata Service
- [x] Unusual Linux Network Activity
- [x] Unusual Linux Network Port Activity
- [x] Anomalous Process For a Linux Population
- [x] Unusual Linux Username
- [x] DNS Tunneling
- [x] Unusual DNS Activity
- [x] Unusual Network Destination Domain Name
- [x] Unusual Web Request
- [x] Unusual Web User Agent
- [x] Network Traffic to Rare Destination Country
- [x] Unusual Process For a Linux Host
- [x] Unusual Process For a Windows Host
- [x] Spike in Network Traffic To a Country
- [x] Unusual Login Activity
- [x] Unusual Windows Process Calling the Metadata Service
- [x] Unusual Windows User Calling the Metadata Service
- [x] Unusual Windows Network Activity
- [x] Unusual Windows Path Activity
- [x] Anomalous Process For a Windows Population
- [x] Anomalous Windows Process Creation
- [x] Suspicious Powershell Script
- [x] Unusual Windows Service
- [x] Unusual Windows Username
- [x] Unusual Windows User Privilege Elevation Activity
- [x] Unusual Windows Remote User
- [x] Adversary Behavior - Detected - Elastic Endgame
- [x] Credential Dumping - Detected - Elastic Endgame
- [x] Credential Dumping - Prevented - Elastic Endgame
- [x] Credential Manipulation - Detected - Elastic Endgame
- [x] Credential Manipulation - Prevented - Elastic Endgame
- [x] Exploit - Detected - Elastic Endgame
- [x] Exploit - Prevented - Elastic Endgame
- [x] Malware - Detected - Elastic Endgame
- [x] Malware - Prevented - Elastic Endgame
- [x] Permission Theft - Detected - Elastic Endgame
- [x] Permission Theft - Prevented - Elastic Endgame
- [x] Process Injection - Detected - Elastic Endgame
- [x] Process Injection - Prevented - Elastic Endgame
- [x] Ransomware - Detected - Elastic Endgame
- [x] Ransomware - Prevented - Elastic Endgame
- [x] External Alerts
- [x] Process Execution from an Unusual Directory
- [x] Execution from Unusual Directory - Command Line
- [x] Suspicious Execution - Short Program Name

</details>

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

File: rules/integrations/aws/exfiltration_rds_snapshot_restored.toml [hunk 1]
```diff
@@ -32,7 +32,7 @@ references = [
 risk_score = 47
 rule_id = "bf1073bf-ce26-4607-b405-ba1ed8e9e204"
 severity = "medium"
-tags = ["Elastic", "Cloud", "AWS", "Continuous Monitoring", "SecOps", "Asset Visibility"]
+tags = ["Elastic", "Cloud", "AWS", "Continuous Monitoring", "SecOps", "Asset Visibility","Defense Evasion"]
```
Comment (w0rk3r) [inline rules/integrations/aws/exfiltration_rds_snapshot_restored.toml:35]:
```suggestion
tags = ["Elastic", "Cloud", "AWS", "Continuous Monitoring", "SecOps", "Asset Visibility", "Defense Evasion"]
```

File: rules/integrations/aws/persistence_redshift_instance_creation.toml [hunk 2]
```diff
@@ -32,7 +32,7 @@ risk_score = 21
 rule_id = "015cca13-8832-49ac-a01b-a396114809f6"
 severity = "low"
 timestamp_override = "event.ingested"
-tags = ["Elastic", "Cloud", "AWS", "Continuous Monitoring", "SecOps", "Asset Visibility"]
+tags = ["Elastic", "Cloud", "AWS", "Continuous Monitoring", "SecOps", "Asset Visibility","Persistence"]
```
Comment (w0rk3r) [inline rules/integrations/aws/persistence_redshift_instance_creation.toml:35]:
```suggestion
tags = ["Elastic", "Cloud", "AWS", "Continuous Monitoring", "SecOps", "Asset Visibility", "Persistence"]
```

File: rules/integrations/azure/impact_azure_automation_runbook_deleted.toml [hunk 3]
```diff
@@ -27,11 +27,20 @@ references = [
...
+    event.outcome:(Success or success)
 '''
 
+[[rule.threat]]
+framework = "MITRE ATT&CK"
+
+[rule.threat.tactic]
+reference = "https://attack.mitre.org/tactics/TA0005/"
+name = "Defense Evasion"
+id = "TA0005"
```
Comment (w0rk3r) [inline rules/integrations/azure/impact_azure_automation_runbook_deleted.toml:46]: We would need to rename this one or maintain the change it to the impact tactic

File: rules/integrations/azure/defense_evasion_azure_automation_runbook_deleted.toml [hunk 4]
```diff
@@ -1,14 +1,14 @@
...
-updated_date = "2021/07/20"
+updated_date = "2022/07/13"
 integration = "azure"
 
 [rule]
 author = ["Elastic"]
 description = """
 Identifies when an Azure Automation runbook is deleted. An adversary may delete an Azure Automation runbook in order to
-disrupt their target's automated business operations or to remove a malicious runbook that was used for persistence.
+disrupt their target's automated business operations or to remove a malicious runbook for defense evasion.
```
Comment (Samirbous) [inline rules/integrations/azure/defense_evasion_azure_automation_runbook_deleted.toml:11]: `to remove a malicious runbook ...` sounds like the customer has it installed by default and the adversary will remove it for defense evasion ?

File: rules/integrations/o365/microsoft_365_teams_custom_app_interaction_allowed.toml [hunk 5]
```diff
@@ -29,7 +29,8 @@ references = ["https://docs.microsoft.com/en-us/microsoftteams/platform/concepts
 risk_score = 47
 rule_id = "bbd1a775-8267-41fa-9232-20e5582596ac"
 severity = "medium"
-tags = ["Elastic", "Cloud", "Microsoft 365", "Continuous Monitoring", "SecOps", "Configuration Audit"]
+tags = ["Elastic", "Cloud", "Microsoft 365", "Continuous Monitoring", "SecOps",
+    "Configuration Audit", "Persistence"]
```
Comment (Samirbous) [inline rules/integrations/o365/microsoft_365_teams_custom_app_interaction_allowed.toml:33]:
```suggestion
tags = ["Elastic", "Cloud", "Microsoft 365", "Continuous Monitoring", "SecOps", "Configuration Audit", "Persistence"]
```

File: rules/promotions/privilege_escalation_endgame_permission_theft_detected.toml [hunk 6]
```diff
@@ -19,10 +19,17 @@ name = "Permission Theft - Detected - Elastic Endgame"
...
 severity = "high"
-tags = ["Elastic", "Elastic Endgame"]
+tags = ["Elastic", "Elastic Endgame", "Threat Detection", "Privilege Escalation"]
 type = "query"
 
 query = '''
 event.kind:alert and event.module:endgame and endgame.metadata.type:detection and (event.action:token_protection_event or endgame.event_subtype_full:token_protection_event)
 '''
 
+[[rule.threat]]
```
Comment (Samirbous) [inline rules/promotions/privilege_escalation_endgame_permission_theft_detected.toml:29]: maybe https://attack.mitre.org/techniques/T1134/ is more accurate ?

File: rules/promotions/privilege_escalation_endgame_permission_theft_prevented.toml [hunk 7]
```diff
@@ -19,10 +19,17 @@ name = "Permission Theft - Prevented - Elastic Endgame"
...
 severity = "medium"
-tags = ["Elastic", "Elastic Endgame"]
+tags = ["Elastic", "Elastic Endgame", "Threat Detection", "Privilege Escalation"]
 type = "query"
 
 query = '''
 event.kind:alert and event.module:endgame and endgame.metadata.type:prevention and (event.action:token_protection_event or endgame.event_subtype_full:token_protection_event)
 '''
 
+[[rule.threat]]
```
Comment (Samirbous) [inline rules/promotions/privilege_escalation_endgame_permission_theft_prevented.toml:29]: maybe https://attack.mitre.org/techniques/T1134/ is more accurate ?

File: rules/windows/execution_from_unusual_directory.toml [hunk 8]
```diff
@@ -45,3 +45,14 @@ process.executable : ("C:\\PerfLogs\\*.exe","C:\\Users\\Public\\*.exe","C:\\User
  /* and not (process.code_signature.subject_name == "Microsoft Corporation" and process.code_signature.trusted == true) */
 '''
 
+[[rule.threat]]
```
Comment (Samirbous) [inline rules/windows/execution_from_unusual_directory.toml:48]: maybe https://attack.mitre.org/techniques/T1036/005/ is more appropriate ?

File: rules/windows/execution_from_unusual_path_cmdline.toml [hunk 9]
```diff
@@ -121,3 +121,14 @@ process where event.type in ("start", "process_started", "info") and
   not (process.name : "rundll32.exe" and process.args : ("uxtheme.dll,#64", "PRINTUI.DLL,PrintUIEntry"))
 '''
 
+[[rule.threat]]
```
Comment (Samirbous) [inline rules/windows/execution_from_unusual_path_cmdline.toml:124]: maybe add also https://attack.mitre.org/techniques/T1036/005/ + T1059 (existing)

File: rules/windows/execution_suspicious_short_program_name.toml [hunk 10]
```diff
@@ -30,3 +30,14 @@ process where event.type in ("start", "process_started") and length(process.name
  length(process.name) == 5 and host.os.name == "Windows" and length(process.pe.original_file_name) > 5
 '''
 
+[[rule.threat]]
```
Comment (Samirbous) [inline rules/windows/execution_suspicious_short_program_name.toml:33]: maybe https://attack.mitre.org/techniques/T1036/003/ is more appropriate ?

**Review by Samirbous — CHANGES_REQUESTED** [review]
nice work!, added few suggestion, in addition to that some rule fille names need to be renamed to pass the test (e.g. tag persistence but rule file name starts with linux_something.toml)

File: rules/integrations/azure/defense_evasion_azure_automation_runbook_deleted.toml (hunk 4 above)
Comment (terrancedejesus) [reply rules/integrations/azure/defense_evasion_azure_automation_runbook_deleted.toml:11]: Good point. Change looks implemented.

File: rules/integrations/google_workspace/google_workspace_policy_modified.toml [hunk 11]
```diff
@@ -39,7 +39,8 @@ The Google Workspace Fleet integration, Filebeat module, or similarly structured
 risk_score = 47
 rule_id = "a99f82f5-8e77-4f8b-b3ce-10c0f6afbc73"
 severity = "medium"
-tags = ["Elastic", "Cloud", "Google Workspace", "Continuous Monitoring", "SecOps", "Identity and Access"]
+tags = ["Elastic", "Cloud", "Google Workspace", "Continuous Monitoring", "SecOps",
+  "Identity and Access", "Persistence"]
```
[the remark this answers is not in the document]
Comment (terrancedejesus) [reply rules/integrations/google_workspace/google_workspace_policy_modified.toml:43]: Resolved.

File: rules/integrations/google_workspace/mfa_disabled_for_google_workspace_organization.toml [hunk 12]
```diff
@@ -39,11 +39,25 @@ The Google Workspace Fleet integration, Filebeat module, or similarly structured
 risk_score = 47
 rule_id = "e555105c-ba6d-481f-82bb-9b633e7b4827"
 severity = "medium"
-tags = ["Elastic", "Cloud", "Google Workspace", "Continuous Monitoring", "SecOps", "Identity and Access"]
+tags = ["Elastic", "Cloud", "Google Workspace", "Continuous Monitoring", "SecOps",
+  "Identity and Access", "Persistence"]
```
[the remark this answers is not in the document]
Comment (terrancedejesus) [reply rules/integrations/google_workspace/mfa_disabled_for_google_workspace_organization.toml:43]: Resolved.

File: rules/integrations/o365/microsoft_365_exchange_dkim_signing_config_disabled.toml [hunk 13]
```diff
@@ -32,11 +32,24 @@ references = [
 risk_score = 47
 rule_id = "514121ce-c7b6-474a-8237-68ff71672379"
 severity = "medium"
-tags = ["Elastic", "Cloud", "Microsoft 365", "Continuous Monitoring", "SecOps", "Data Protection"]
+tags = ["Elastic", "Cloud", "Microsoft 365", "Continuous Monitoring", "SecOps",
+    "Data Protection", "Persistence"]
```
[the remark this answers is not in the document]
Comment (terrancedejesus) [reply rules/integrations/o365/microsoft_365_exchange_dkim_signing_config_disabled.toml:36]: Resolved.

File: rules/integrations/o365/microsoft_365_teams_custom_app_interaction_allowed.toml (hunk 5 above)
Comment (terrancedejesus) [reply rules/integrations/o365/microsoft_365_teams_custom_app_interaction_allowed.toml:33]: Resolved.

File: rules/promotions/privilege_escalation_endgame_permission_theft_detected.toml (hunk 6 above)
Comment (terrancedejesus) [reply rules/promotions/privilege_escalation_endgame_permission_theft_detected.toml:29]: added as the technique.

File: rules/windows/execution_from_unusual_directory.toml (hunk 8 above)
Comment (terrancedejesus) [reply rules/windows/execution_from_unusual_directory.toml:48]: agreed, adjusted.

File: rules/windows/execution_suspicious_short_program_name.toml (hunk 10 above)
Comment (terrancedejesus) [reply rules/windows/execution_suspicious_short_program_name.toml:33]: adjusted.

File: rules/promotions/credential_access_endgame_cred_dumping_detected.toml [hunk 14]
```diff
@@ -19,10 +19,22 @@ name = "Credential Dumping - Detected - Elastic Endgame"
...
 event.kind:alert and event.module:endgame and endgame.metadata.type:detection and (event.action:cred_theft_event or endgame.event_subtype_full:cred_theft_event)
 '''
 
+[[rule.threat]]
+framework = "MITRE ATT&CK"
+[[rule.threat.technique]]
+id = "T1003"
+name = "OS Credential Dumping"
+reference = "https://attack.mitre.org/techniques/T1003/"
+
```
[the remark this answers is not in the document]
Comment (terrancedejesus) [reply rules/promotions/credential_access_endgame_cred_dumping_detected.toml:35]: thanks for the insight, adjusted

File: rules/promotions/credential_access_endgame_cred_dumping_prevented.toml [hunk 15]
```diff
@@ -19,10 +19,21 @@ name = "Credential Dumping - Prevented - Elastic Endgame"
...
 severity = "medium"
-tags = ["Elastic", "Elastic Endgame"]
+tags = ["Elastic", "Elastic Endgame", "Threat Detection", "Credential Access"]
 type = "query"
 
 query = '''
 event.kind:alert and event.module:endgame and endgame.metadata.type:prevention and (event.action:cred_theft_event or endgame.event_subtype_full:cred_theft_event)
 '''
 
+[[rule.threat]]
```
[the remark this answers is not in the document]
Comment (terrancedejesus) [reply rules/promotions/credential_access_endgame_cred_dumping_prevented.toml:29]: adjusted

File: rules/promotions/persistence_endgame_cred_manipulation_detected.toml [hunk 16]
```diff
@@ -19,10 +19,23 @@ name = "Credential Manipulation - Detected - Elastic Endgame"
...
 severity = "high"
-tags = ["Elastic", "Elastic Endgame"]
+tags = ["Elastic", "Elastic Endgame", "Threat Detection", "Persistence"]
 type = "query"
 
 query = '''
 event.kind:alert and event.module:endgame and endgame.metadata.type:detection and (event.action:token_manipulation_event or endgame.event_subtype_full:token_manipulation_event)
 '''
 
+[[rule.threat]]
```
[the remark this answers is not in the document]
Comment (terrancedejesus) [reply rules/promotions/persistence_endgame_cred_manipulation_detected.toml:29]: adjusted

**Comment by terrancedejesus** [comment]
The following rules have min_stack_required and received an adjustment in this PR. Because of this, we need to pay attention to versions when we lock for FF as some changes will not be accounted for between branches.

- Domain Added to Google Workspace Trusted Domains - 8.0
- Google Workspace MFA Enforcement Disabled - 8.0
- Google Workspace Password Policy Modified - 8.0
- MFA Disabled for Google Workspace Organization - 8.0

Machine Learning
- Unusual Linux Process Calling the Metadata Service - 8.3.0
- Unusual Linux User Calling the Metadata Service - 8.3.0
- Unusual Login Activity - 8.3.0
- Unusual Windows Process Calling the Metadata Service - 8.3.0
- Unusual Windows User Calling the Metadata Service - 8.3.0
- Unusual Linux System Information Discovery Activity - 8.3.0
- Unusual Linux System Network Configuration Discovery - 8.3.0
- Unusual Linux Network Connection Discovery - 8.3.0
- Unusual Linux Process Discovery Activity - 8.3.0
- Unusual Linux System Owner or User Discovery Activity - 8.3.0
- Suspicious Powershell Script - 8.3.0
- Unusual Linux Username - 8.3.0
- Unusual Windows Username - 8.3.0
- Unusual Windows Remote User - 8.3.0
- Anomalous Process For a Linux Population - 8.3.0
- Unusual Process For a Linux Host - 8.3.0
- Unusual Process For a Windows Host - 8.3.0
- Unusual Windows Path Activity - 8.3.0
- Anomalous Process For a Windows Population - 8.3.0
- Anomalous Windows Process Creation - 8.3.0
- Unusual Windows Service - 8.3.0
- Unusual Sudo Activity - 8.3.0
- Unusual Windows User Privilege Elevation Activity - 8.3.0
- Anomalous Linux Compiler Activity - 8.3.0

Of these rules, the following rules have been adjusted and and because of the new att&ck mapping version, had technique name changes. Since these are min stacked we may have to manually go to these branches and update the names since the updated att&ck mapping will be backported to all currently supported branches.

- Application Added to Google Workspace Domain - 8.0
- ImageLoad via Windows Update Auto Update Client - 8.2

**Comment by Mikaayenson** [comment]
@spong Do you have to update mappings on your end?

**Review by imays11 — APPROVED** [review]
Looks good!

File: rules/integrations/o365/persistence_microsoft_365_teams_custom_app_interaction_allowed.toml [hunk 17]
```diff
@@ -40,3 +41,9 @@ o365.audit.Name:"Allow sideloading and interaction of custom apps" and
 o365.audit.NewValue:True and event.outcome:success
 '''
 
+[[rule.threat]]
+framework = "MITRE ATT&CK"
+[rule.threat.tactic]
+id = "TA0003"
+name = "Persistence"
+reference = "https://attack.mitre.org/tactics/TA0003/"
```
Comment (Mikaayenson) [inline rules/integrations/o365/persistence_microsoft_365_teams_custom_app_interaction_allowed.toml:49]: \n needed

File: rules/integrations/okta/defense_evasion_okta_attempt_to_delete_okta_policy.toml [hunk 18]
```diff
@@ -32,11 +32,29 @@ references = [
...
+  [[rule.threat.technique.subtechnique]]
+  id = "T1562.007"
+  name = "Disable or Modify Cloud Firewall"
+  reference = "https://attack.mitre.org/techniques/T1562/007/"
+
+
+[rule.threat.tactic]
+id = "TA0005"
+name = "Defense Evasion"
+reference = "https://attack.mitre.org/tactics/TA0005/"
```
Comment (Mikaayenson) [inline rules/integrations/okta/defense_evasion_okta_attempt_to_delete_okta_policy.toml:60]: \n needed

**Review by Mikaayenson — APPROVED** [review]
This LGTM. Great work on updating the mappings. As you mentioned [here](https://github.com/elastic/detection-rules/pull/2073#issuecomment-1191638959):
- We are accepting that rules on older branches will fail with this new attack mapping and will have to address them separately.


One minor thing: a few rules look like the need `\n` added.

**Comment by terrancedejesus** [comment]
@Samirbous I have addressed your comments. Thanks for the feedback! Once you approve, I will merge this one in and address the manual backport issues with two specific rules.

**Comment by spong** [comment]
> @spong Do you have to update mappings on your end?

Yeah, we'll need to on the Kibana side as well thanks for the ping! I'm currently on leave, @banderror can you take care of this please? Here's a PR where we've done it in the past -- just need to update one file and run a script: https://github.com/elastic/kibana/issues/89876

**Comment by banderror** [comment]
@spong Of course, will do it on Monday!

Btw @spong is on leave till November, so please ping me instead or our area in `#security-detection-rules-area` (@elastic/security-detections-response-rules).

**Review by Samirbous — APPROVED** [review]

**Comment by banderror** [comment]
@terrancedejesus @Mikaayenson I don't know if you'd like to review the PR, but here it is:
https://github.com/elastic/kibana/pull/137122

**Comment by terrancedejesus** [comment]
> @terrancedejesus @Mikaayenson I don't know if you'd like to review the PR, but here it is:
> 
> https://github.com/elastic/kibana/pull/137122

Hey there! Thanks for linking the PR. I don't think we need to review it at the moment as we already merged our changes. As a side note we are using tag 11.3 from ATT&CK so as long as those align we should be good!

**Comment by banderror** [comment]
Oh, that's actually good to know @terrancedejesus, because we were pulling the MITRE content from their `master` branch. I updated the script to pull from the specific version and made sure we use `v11.3` in `8.4.0`: https://github.com/elastic/kibana/pull/137122/commits/1bda53f0765bd3f8a8a1b5e429663abaf40f2faf. Thank you!

**Pull request merged**